### PR TITLE
fix(lpa-questionnaire-web-app): Invalid files are nolonger break muli…

### PR DESCRIPTION
**Pull request by LouiseMcMahon**
…file upload

## Ticket Number

https://pins-ds.atlassian.net/browse/AS-

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js [hunk 1]
```diff
@@ -204,6 +203,7 @@ Then('any document uploaded will not be saved', () => {
 });
 
 Then('the file is removed', () => {
+  cy.wait(1000);
```
Comment (rcmsoftwaredevelopment) [inline lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js:206]: This line needs to be changed to be in line with standards.

File: lpa-submissions-e2e-tests/cypress/support/common/validateFileUploadErrorMessage.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/// <reference types = "Cypress"/>
```
Comment (rcmsoftwaredevelopment) [inline lpa-submissions-e2e-tests/cypress/support/common/validateFileUploadErrorMessage.js:1]: Can this be removed.

File: packages/lpa-questionnaire-web-app/src/views/layouts/multi-file-upload.njk [hunk 3]
```diff
@@ -75,6 +75,7 @@
 
         <input type="hidden" id="tempDocs" name="tempDocs" value="{{ documentList }}" />
 
+
```
Comment (rcmsoftwaredevelopment) [inline packages/lpa-questionnaire-web-app/src/views/layouts/multi-file-upload.njk:78]: This can be removed

**Review by rcmsoftwaredevelopment — CHANGES_REQUESTED** [review]

File: lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js (hunk 1 above)
Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js:206]: This is required as there is a ms difference in the file getting deleted and updated

File: lpa-submissions-e2e-tests/cypress/support/common/validateFileUploadErrorMessage.js (hunk 2 above)
Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/support/common/validateFileUploadErrorMessage.js:1]: This is currently consistent with what we have on the project. I can take this as a separate task to remove it

**Review by rcmsoftwaredevelopment — APPROVED** [review]

File: lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js [hunk 4]
```diff
@@ -193,9 +193,8 @@ Then('progress is halted with a message the file {string} {string}', (fileName,
       break;
   }
 
-  cy.validateErrorMessage(errorMessage, null, fileName);
-  validateFileUpload(errorMessage);
-  verifyUploadPageTitleError();
+  cy.validateFileUploadErrorMessage(errorMessage, null);
+  // validateFileUpload(errorMessage);
```
[the remark this answers is not in the document]
Comment (shilpabasu22) [reply lpa-submissions-e2e-tests/cypress/integration/common/uploadFiles.js]: Updated as per comments

**Review by rcmsoftwaredevelopment — CHANGES_REQUESTED** [review]

File: packages/lpa-questionnaire-web-app/src/views/layouts/multi-file-upload.njk [hunk 5]
```diff
@@ -112,7 +112,7 @@ if (typeof MOJFrontend.MultiFileUpload !== 'undefined') {
       error: response.error ? response.error.message : undefined,
     };
 
-    temporaryDocInput.value = temporaryDocInput.value + JSON.stringify(upload);
+      temporaryDocInput.value = temporaryDocInput.value + JSON.stringify(upload);
```
[the remark this answers is not in the document]
Comment (LouiseMcMahon) [reply packages/lpa-questionnaire-web-app/src/views/layouts/multi-file-upload.njk]: Its just a left over white space change as a fix was tested here

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Pull request merged**